src/sentry/models/latestappconnectbuildscheck.py

```diff
@@ -0,0 +1,47 @@
+"""Database models to keep track of the App Store Connect builds for a project.
+
+If a project enables the App Store Connect source to download dSYMs directly from Apple we
+need to keep track of which builds have already been downloaded.
+"""
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model
+
+
+class LatestAppConnectBuildsCheck(Model):
+    """
+    The last date and time Sentry checked App Store Connect for new builds associated with a
+    specific appconnect source in a project.
+    """
+
+    __include_in_export__ = False
+
+    project = FlexibleForeignKey("sentry.Project", db_constraint=False)
+
+    # The symbol source's unique identifier in the project. Also known as the id on an
+    # AppStoreConnectConfig.
+    source_id = models.CharField(max_length=200, default="")
+
+    # When sentry last checked App Store Connect.
+    last_fetched = models.DateTimeField(default=timezone.now)
+
+    @classmethod
+    def refresh_date(cls, project, source_id):
```

Comment:
rather low priority nit: i find "refresh" somewhat confusing and would choose "update" myself.  No refresh action is being done, a row is being updated.  🤷